shell/src/main/java/tachyon/shell/TFsShell.java

```diff
@@ -812,16 +745,21 @@ public int tail(TachyonURI path) throws IOException {
    * Creates a 0 byte file specified by argv.
    *
    * @param path The TachyonURI path as the input of the command
-   * @return 0 if command if successful, -1 if an error occurred.
    * @throws IOException
    */
-  public int touch(TachyonURI path) throws IOException {
+  public void touch(TachyonURI path) throws IOException {
```

Comment:
Wasn't sure how to test failing after the `createFile` call here, and I couldn't find a hacky way to do it like I did for `copyFromLocal`, any thoughts?